tests/Antlers/ParserTestCase.php

```diff
@@ -229,6 +229,7 @@ protected function renderString($text, $data = [], $withCoreTagsAndModifiers = f
         $processor->setData($data);
 
         $runtimeParser = new RuntimeParser($documentParser, $processor, new AntlersLexer(), new LanguageParser());
+        $processor->setAntlersParserInstance($runtimeParser);
```

Comment:
This just ensures that the environment eventually gets a Parser instance when it goes to resolve the `antlers: true` content in the new test coverage.